src/include/libabrt.h

```diff
@@ -44,7 +44,7 @@ void abrt_ensure_writable_dir_uid_gid(const char *dir, mode_t mode, uid_t uid, g
 void abrt_ensure_writable_dir(const char *dir, mode_t mode, const char *user);
 void abrt_ensure_writable_dir_group(const char *dir, mode_t mode, const char *user, const char *group);
 char *abrt_run_unstrip_n(const char *dump_dir_name, unsigned timeout_sec);
-char *abrt_get_backtrace(const char *dump_dir_name, unsigned timeout_sec, const char *debuginfo_dirs);
+char *abrt_get_backtrace(struct dump_dir *dd, const char *dump_dir_name, unsigned timeout_sec, const char *debuginfo_dirs);
```

Comment:
Passing `dump_dir_name` is redundant now since it can be obtained from `dd`, isn'it?